.github/workflows/linux-tests.yml

```
@@ -20,6 +20,7 @@ jobs:
       fail-fast: false 
       matrix:
         PYTHON_VERSION: ['3.8', '3.10']
+        SPYDER_SOURCE: ['conda', 'git']
```

Lets for the moment only set here `git` since Spyder>=6.0  or even Spyder>=6.0.0a3 is not available yet:
```suggestion
        SPYDER_SOURCE: ['git']
```

---

Also the name config for the job a couple of lines above (around line 13) is missing the addition of the source info. The `name` config should be something like:

```yaml
name: Linux Py${{ matrix.PYTHON_VERSION }} spyder-from-${{ matrix.SPYDER_SOURCE }}
```